tools/tasks/gulp-pa11y.js

```diff
@@ -0,0 +1,92 @@
+const fs = require('fs');
+const path = require('path');
+
+const pa11y = require('pa11y');
+const reporter = require('pa11y-reporter-cli'); // As pa11y 5 stabilizes, we can pull this off pa11y
+const _ = require('lodash');
+const through = require('through2');
+
+// Grab project root pa11y.json as object
+const pallyJson = JSON.parse(fs.readFileSync(path.join(process.cwd(), 'pa11y.json')));
+// Custom logging tools for pa11y
```

Comment:
Why do we need custom logging tools? Shouldn't CLI logging be part of why we are using the CLI reporter?

Reply:
This is boilerplate pa11y and I'd definitely like to look into cleaning this up in the future. For instance, you'll notice that we do this to output log info:

```shell
console.log(reporter.results(result));
```

With more research, I'm guessing it's actually more like:

```shell
pa11y.log.info(reporter.results(result));
```

Reply:
✅

Reply:
Updated.